Homework 6

TaraShevchenko opened the pull request:
Не справился с 3-м заданием. Не понял как запушить какой то обьтект в бд

Commit:
Homework 6

ZarMarathon reviewed (commented):
доделай по моим комментам

Commit:
homework6.1

Исправил все ощибки. все и правда оказалось не так и сложно. Не знаю чего так тупил )

Commit:
homework7.1
